packages/ndla-ui/src/ContactBlock/ContactBlock.tsx

```diff
@@ -1,160 +1,193 @@
 /**
- * Copyright (c) 2023-present, NDLA.
+ * Copyright (c) 2024-present, NDLA.
  *
  * This source code is licensed under the GPLv3 license found in the
  * LICENSE file in the root directory of this source tree.
  *
  */
 
-/** @jsxImportSource @emotion/react */
 import { ReactNode } from "react";
 import { useTranslation } from "react-i18next";
-import { css } from "@emotion/react";
-import styled from "@emotion/styled";
-import { spacing, fonts, colors, mq, breakpoints, misc } from "@ndla/core";
-import { BlobPointy as UnstyledBlobPointy, BlobRound as UnstyledBlobRound } from "@ndla/icons/common";
 import { COPYRIGHTED, getLicenseByAbbreviation } from "@ndla/licenses";
-import { Image } from "@ndla/primitives";
+import { Text } from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
 import { IAuthor, IImageMetaInformationV3 } from "@ndla/types-backend/image-api";
 import { CanonicalUrlFuncs } from "../Embed";
 import { errorSvgSrc } from "../Embed/ImageEmbed";
 import LicenseLink from "../LicenseByline/LicenseLink";
 
-const BLOB_WIDTH = 90;
+export const contactBlockBackgrounds = ["strong", "moderate", "subtle"] as const;
+export type ContactBlockBackground = (typeof contactBlockBackgrounds)[number];
```

Comment:
Man kan bruke panda-typer for å holde dette i sync med variant-typen.

`export type ContactBlockBackground = NonNullable<StyledVariantProps<typeof StyledWrapper>["variant"]>;`

Reply:
Hvis du syns det fungerer å dra variants ut i et eget objet sånn jeg har gjort nå så tror jeg det løser det :smile: